package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -115,10 +115,10 @@
 .. Links
 
 .. _AMBER: http://ambermd.org
-.. _AMBER TRJ format: http://ambermd.org/formats.html#trajectory
+.. _AMBER TRJ format: http://www.webcitation.org/query?url=http%3A%2F%2Fambermd.org%2Fformats.html&date=2018-02-11 
```

Comment:
Tough when we have to go to archived pages, but that's a good way to replace the link.

IIRC, when I used to find a format page I submitted a webcitation request for it, too, assuming that the pages would eventually go away. 

Scary if the top page in a web search for a file format is the MDAnalysis reader...

Reply:
That new link doesn't work for me.

I'm not sure why we're replacing the old one? This remains the official documentation for the amber non-binary trajectory format: https://ambermd.org/FileFormats.php#trajectory

Is there any reason why we're not u sing that?

Reply:
```suggestion
.. _AMBER TRJ format: https://ambermd.org/FileFormats.php#trajectory
```

Reply:
Thanks for finding the original one on the Amber website.